core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/sort.kt

```diff
@@ -67,4 +72,27 @@ class SortDataColumn {
         col.sortWith { df1, df2 -> df1[a] - df2[a] } shouldBe sortedCol
         col.sortWith(compareBy { it[a] }) shouldBe sortedCol
     }
+
+    @Test
```

Comment:
Cool new dataset for unit tests! Thanks!

Reply:
Great to validate on that some high-level integration tests with more complex logic instead of low-level A.a.b